spec/support/shared_examples_policy_machine.rb

```diff
@@ -220,6 +220,33 @@
     end
   end
 
+  describe 'Operations' do
+
+    it 'does not allow an operation to start with a ~' do
+      expect{policy_machine.create_operation('~apple')}.to raise_error(ArgumentError)
+      expect{policy_machine.create_operation('apple~')}.not_to raise_error
+    end
+
+    it 'can derive a prohibition from an operation and vice versa' do
+      @op = policy_machine.create_operation('fly')
+      expect(@op.prohibition).to be_prohibition
+      expect(@op.prohibition.operation).to eq('fly')
+    end
+
+    it 'raises if trying to negate a non-operation' do
+      expect{PM::Prohibition.on(3)}.to raise_error(ArgumentError)
+    end
+
+    it 'can negate operations expressed as strings' do
+      expect(PM::Prohibition.on('fly')).to be_a String
+    end
+
+    it 'can negate operations expressed as symbols' do
```

Comment:
Should there be another test that it can negate operations of class Operation per https://github.com/mdsol/the_policy_machine/pull/40/files#diff-cd736b91a0f9140de3bb79b73ec78b70R308